extension.neon

```diff
@@ -24,14 +18,3 @@ services:
 		tags:
 			- phpstan.broker.dynamicMethodReturnTypeExtension
 		autowired: false
```

Comment:
I left `ProphesizeDynamicReturnTypeExtension` to avoid re-defining a stub of PHPUnit's `TestCase`, due to both #161 and to avoid collisions with `phpstan/phpstan-phpunit`.